backend/core/src/entity/listing.entity.ts

```diff
@@ -108,6 +113,11 @@ class Listing extends BaseEntity {
   urlSlug?: string
   @OneToMany((type) => Application, (application) => application.listing)
   applications: Application[]
+  @Column({
+    type: "enum",
+    enum: ListingStatus,
```

Comment:
Should we add a default here at the database level to make creation easier?

Reply:
Sure.